src/components/Error/Error.tsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, FC } from 'react';
+
+/* eslint-disable jsx-a11y/control-has-associated-label */
+type Props = {
+  errorText: string,
+  errorClear: () => void
+};
+export const Error: FC<Props> = ({ errorClear, errorText }) => {
+  useEffect(() => {
+    const idTimer = setTimeout(errorClear, 3000);
+
+    if (errorText) {
+      // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+      idTimer;
+    }
```

Comment:
can we remove this?

Reply:
i try but linter Swearing Expected an assignment or function call and instead saw an expression, and since it is a timer I can not call it as a function other than to ignore this line for him I could not find, well, unless you go to the settings linter but I think this is not a good idea

Reply:
What if you use like this? Just as an attempt
```
useEffect(() => {
  let idTimer: ReturnType<typeof setTimeout>;

  if (errorText) {
    idTimer = setTimeout(errorClear, 3000);
  }

  return () => clearTimeout(idTimer);
}, [errorText, errorClear]);
```

or even like this

```
const clearError = () => {
  if (errorText) {
    errorClear();
  }
};

useEffect(() => {
  const idTimer = setTimeout(clearError, 3000);

  return () => clearTimeout(idTimer);
}, [errorText, errorClear]);
```